aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/ExecutorStepTest.java

```diff
@@ -66,11 +70,16 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
+import static org.hamcrest.Matchers.*;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStep;
+import org.jenkinsci.plugins.workflow.support.steps.ExecutorStepExecution;
```

Comment:
Nope. These classes belong to <code>org.jenkinsci.plugins.workflow.SUPPORT</code>, so they are not visible by default